pkgs/tools/security/softhsm/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, unzip, botan }:
+
+stdenv.mkDerivation rec {
+
+  name = "softhsm-${version}";
+  majorVersion = "2";
+  version = "${majorVersion}.0.0";
+
+  src = fetchurl {
+    url = "https://dist.opendnssec.org/source/${name}.tar.gz";
+    sha256 = "eae8065f6c472af24f4c056d6728edda0fd34306f41a818697f765a6a662338d";
+  };
+
+  configureFlags = [
+    "--with-crypto-backend=botan"
+    "--with-botan=${botan}"
+    "--sysconfdir=$out/etc"
```

Comment:
Would this not make configuration impossible? $out/etc is mounted readonly usually.